nimbus.fml.yaml

```diff
@@ -0,0 +1,134 @@
+---
+channels:
+  - Fennec
+  - FirefoxBeta
+  - Firefox
+features:
+  search:
+    description: The search feature
+    variables:
+      awesome-bar:
+        description: "This property is covers the properties related to the awesome-bar"
+        type: AwesomeBar
+        default:
+          {
+            "use-page-content": false
+          }
+      spotlight:
+        description: "This property is covers the properties related to the spotlight"
+        type: Spotlight
+        default:
+          {
+            "enabled": false,
+            "description": "",
+            "use-html-content": true,
+            "icon": "letter",
+            "keep-for-days": 100
+          }
+    defaults: null
+  nimbus-validation:
+    description: "A feature that does not correspond to an application feature suitable for showing
+    that Nimbus is working."
+    variables:
+      settings-title:
+        description: The title of displayed in the Settings screen and app menu.
+        type: String
+        default: "app_menu_settings_title"
+      settings-title-punctuation:
+        description: The emoji displayed in the Settings screen title.
+        type: String
+        default: ""
+      settings-icon:
+        description: The drawable displayed in the app menu for Settings
+        type: String
+        default: "menu-Settings"
+  homescreenFeature:
+    description: The homescreen that the user goes to when they press home or new tab.
+    variables:
+      sections-enabled:
+        description: "This property provides a lookup table of whether or not the given section should be enabled.
+        If the section is enabled, it should be toggleable in the settings screen, and on by default."
+        type: Map<HomeScreenSection, Boolean>
+        default:
+          {
+            "top-sites": true,
+            "jump-back-in": false,
+            "recently-saved": false,
+            "recent-explorations": false,
+            "pocket": false,
+            "library-shortcuts": false
```

Comment:
Question for @nishant2718 : should these all be true now?

Reply:
I'm not sure what `recent-explorations` is to be honest 😅 . We can probably keep that false for all channels for now. 

In Prod, there's currently an experiment running that enables Jump Back In and Recently Saved. In Beta, both of those are default off.

So maybe we need two additional channel defaults for Prod and Beta that reflect that? Where Prod and Beta would have those two off (and the rest on).

Reply:
FWIW, the values here are what I set as I was going over what experiments are in use in iOS - we'll need to make sure they're correct when iOS switches its experiments to the FML completely - but for now, having the FML should unblock the messaging work

I'll follow up with another PR to switch the experiments we have to use the FML, and we can correct those values there